app/menu.js

```diff
@@ -112,21 +135,19 @@ var createDefaultMenu = function(app, checkForUpdates) {
                   focusedWindow.setFullScreen(!focusedWindow.isFullScreen());
                 }
               }
-            }
-          ]
-        },
-        {
-          label: 'Window',
-          submenu: [
+            },
             {
-              label: 'Minimize',
-              accelerator: 'Command+M',
-              role: 'minimize'
+              type: 'separator'
             },
             {
-              label: 'Close',
-              accelerator: 'Command+W',
-              role: 'close'
+              label: 'Main Window',
+              accelerator: 'Command+1',
+              click: function() {
+                var mainWindow = BrowserWindow.fromId(1);
```

Comment:
Another case where we shouldn't assume `1` I don't think—I'll pass the main window into here.